### DISCUSSION TITLE
Snapshot request errors

### DISCUSSION THREAD

**Discussion by mfregonassi** [tsightler/ring-mqtt#584] [Support Q&A]
I am getting snapshot request errors for the cameras frequently

2023-03-11T16:56:33.524Z ring-mqtt [camera.garden.01] Requesting an updated interval snapshot
2023-03-11T16:56:38.687Z ring-mqtt [camera.garden.01] RequestError: getaddrinfo EAI_AGAIN app-snaps.ring.com
2023-03-11T16:56:38.687Z ring-mqtt [camera.garden.01] Failed to retrieve updated snapshot

Are there any recommendations for this? Tried increasing the snapshot intervals (from the default 30s to 60s) to see if querying app-snaps less often would improve it, but I couldn't see a big difference.

I am investigating if this could be one of the reasons I am facing instability in my setup since I added this camera (floodlight pro) making the container hang from time to time. But I am not sure if it's related because I see snapshot request errors for cameras (doorbell pro and stick up cam) there were already there.

**Comment by tsightler** [comment]
> getaddrinfo EAI_AGAIN

This is a DNS error and means that your DNS server is responding that it was temporarily unable to resolve the DNS entry.  You'll need to troubleshoot the DNS setup to determine the cause.  One thing, NodeJS does not implement any DNS caching at all, so ring-mqtt can generate a huge number of DNS queries and it is really important that you have a local caching DNS server.
